Assets/Scripts/DataController.cs

```diff
@@ -205,22 +201,29 @@ private void Init()
             // get magnitudes and normalize between 1 and 5 to scale stars
             minMag = allStars.Min(s => s.Mag);
             maxMag = allStars.Max(s => s.Mag);
+
             constellationFullNames = new List<string>(allStars.GroupBy(s => s.ConstellationFullName).Select(s => s.First().ConstellationFullName));
-            var constellationNames = new List<ConstellationNamePair>(allStars.GroupBy(s => s.ConstellationFullName).Select(s => new ConstellationNamePair{shortName = s.First().Constellation, fullName = s.First().ConstellationFullName}));
+
+            var constellationNames = new List<ConstellationNamePair>(allStars.GroupBy(s => s.ConstellationFullName).Select(s => new ConstellationNamePair { shortName = s.First().Constellation, fullName = s.First().ConstellationFullName }));
             Debug.Log(minMag + " " + maxMag + " constellations:" + constellationNames.Count);
 
             constellationFullNames.Sort();
+            constellationNames.Add(new ConstellationNamePair { shortName = "test", fullName = "test" });
 
             foreach (ConstellationNamePair constellationName in constellationNames)
             {
+                Debug.Log(constellationName.shortName + " " + constellationName.fullName);
```

Comment:
Might consider commenting out so the console log doesn't get too busy when first running.

Reply:
Removed!